legacy_publisher/dita_helper.py

```diff
@@ -169,3 +264,130 @@ def create_image(root=None,url=None,style=None):
     image.setAttribute('width', width)
     return image
 
+def create_dita_block(root=None,section=None, bullets=None):
```

Comment:
note: I don´t think ´bullets´ is the correct name for the this parameter.  ´content´ or ´children´?